src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -59,7 +62,32 @@
   <md-datepicker
       #resultPicker2
       [touchUi]="touch"
+      [disabled]="datepickerDisabled"
       [startAt]="startAt"
       [startView]="yearView ? 'year' : 'month'">
   </md-datepicker>
 </p>
+
+<h2>Input disabled datepicker</h2>
+<p>
+  <button [mdDatepickerToggle]="datePicker1"></button>
+  <md-input-container>
+    <input mdInput [mdDatepicker]="datePicker1" [(ngModel)]="date" [min]="minDate" [max]="maxDate"
+           [mdDatepickerFilter]="filterOdd ? dateFilter : null" [disabled]="true"
+           placeholder="Input disabled">
+  </md-input-container>
+  <md-datepicker #datePicker1 [touchUi]="touch" [startAt]="startAt"
+                   [startView]="yearView ? 'year' : 'month'"></md-datepicker>
+</p>
+
+<h2>Input disabled, datepicker popup enabled</h2>
+<p>
+  <button [mdDatepickerToggle]="datePicker2"></button>
+  <md-input-container>
+    <input mdInput [mdDatepicker]="datePicker2" [(ngModel)]="date" [min]="minDate" [max]="maxDate"
+           [mdDatepickerFilter]="filterOdd ? dateFilter : null" [disabled]="true"
```

Comment:
Since `disabled` will always be true here, you should be able to just use the attribute.

```
<input mdInput disabled>
```